contrib-04-HOP-LoKi-Functor/ProceedingHOP.md

```diff
@@ -1,9 +1,8 @@
-
 # Introduction
 
-The goal of this project is to create a new LoKi functor and a new TupleTool to implement the ideas of the HOP mass, discussed in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en). This tool is conceived to exploit the kinematic characteristics of $B$ decays into final states involving electrons, by computing a mass in which the bremsstrahlung losses are balanced along the $B$ momentum. The computation of $p_{\mathrm{T}}$ with respect to the $B$ flight direction is already implemented in the LoKi code. This is used later in the LoKi functor implementation of the HOP mass. However, this is not the case for the TupleTool, so this function is implemented as well in the new tool.
+Carried out in the context of the 2016 Impactkit hackathon, this project aims at creating a new LoKi functor and a new TupleTool to implement the ideas of the HOP mass, discussed in [LHCb-INT-2015-037](https://cds.cern.ch/record/2102345?ln=en). This variable is conceived to exploit the kinematic characteristics of $B$ decays into final states involving electrons, by computing a mass in which the bremsstrahlung losses are balanced along the $B$ momentum. It is a variable of interest for all analyses of decays including at the same time electrons and other particles in the final state. In the first part, the algorithm chosen to build the HOP mass will be presented. The precise implementation of this variable as a LoKi functor is described in the second part while the third part focuses on the TupleTool implementation. 
```

Comment:
What about: "aims to implement the concept of the HOP mass, discussed in ..., in the LHCb software to make them more accessible to all the collaboration members"?

Start new paragraph in "In the first part", and put before: "This note documents the algorithm used for the implementation of the HOP variables and also provides a pedagogical overview of how these tools have been implemented. In the first part of this note..."